batch/batch/front_end/front_end.py

```diff
@@ -1034,6 +1019,30 @@
     raise web.HTTPFound(location=location)
 
 
+async def cancel_batch_loop_body(app):
+    async with aiohttp.ClientSession(
+            raise_for_status=True, timeout=aiohttp.ClientTimeout(total=5)) as session:
+        await request_retry_transient_errors(
+            session, 'POST',
+            deploy_config.url('batch-driver', f'/api/v1alpha/batches/cancel'),
+            headers=app['driver_headers'])
+
+    # wait
```

Comment:
What are the wait comments for?

Reply:
To remind the reader that the return value indicates that the run_if_changed logic should wait.

I've forgotten the return value in the past, so I just wanted to emphasize it.

What is really want is named return values so I can document the return value by giving it the name the caller expects, something like: `return should_wait=True` which would correspond to this line in run_if_changed:

 >         should_wait = await f(*args, **kwargs)

but of course Python doesn't let you do that.

Reply:
I could change it to:

```
  should_wait = True
  return should_wait
```

Reply:
I changed this to be the documentation-through-code style.